tests/devices/test_null_qubit.py

```diff
@@ -63,7 +62,7 @@ def test_debugger_attribute():
     assert dev._debugger is None
 
 
-def test_resource_tracking_attribute():
+def test_resource_tracking_attribute(capsys):
```

Comment:
```suggestion
def test_resource_tracking_attribute():
```

Reply:
Capsys is required here as a test fixture since we need to capture stdout to process the resources that are printed